ruleset/rules/0998-aws-security-hub-rules.xml

```diff
@@ -0,0 +1,491 @@
+<!--
+  -  AWS Security Hub default ruleset
+  -  Created by Wazuh, Inc.
+  -  Copyright (C) 2024, Wazuh Inc.
+  -  This program is a free software; you can redistribute it and/or modify it under the terms of GPLv2.
+  ID: 99800 - 99860
+-->
+
+<group name="amazon,securityhub,">
```

Comment:
Change group name to `aws_security_hub`. Use only one group name. Similar to the screenshot:
![image](https://github.com/wazuh/wazuh/assets/97736018/63f121f5-e569-4cbe-98de-56d851d7301c)

Reply:
This has been resolved https://github.com/wazuh/wazuh/pull/23234/commits/1d4bfae8c2701e7cc0cda7b3f8df0965bdb225f8.